package.json

```diff
@@ -33,18 +33,20 @@
     "whatwg-fetch": "^2.0.4"
   },
   "devDependencies": {
-    "babel-eslint": "^9.0.0",
+    "babel-eslint": "9.0.0",
     "cpx": "^1.5.0",
     "env-cmd": "^8.0.2",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint-config-prettier": "^3.0.1",
-    "eslint-plugin-prettier": "^2.6.2",
+    "eslint-config-prettier": "3.0.1",
```

Comment:
I had a problem while running `npm install` on my local environment. 
Maybe i was mess something , but with the latest version of `eslint` most of the tests were failing, so i've locked the version in the package.json to prevent this to happen to others.